app/views/profile/edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1  class="govuk-heading-l">Edit Profile</h1>
+
+<div>
+  <h2 class="govuk-heading-m">New Profile Data:</h2>
+  <%= form_for @user, url: user_registration_path, method: :patch, as: :user do |f| %>
+  <div class="govuk-form-group">
+      <%= f.label :user_id, "User ID", class: "govuk-label" %>
+      <% if @user.user_id == '00000000-0000-0000-0000-000000000000' %>
+        <%= f.text_field :user_id, value: @user.user_id,  class: "govuk-input" %>
+      <% else %>
+        <%= f.text_field :user_id, value: @user.user_id, disabled: true, 
+        class: "govuk-input govuk-input--disabled", 
+        style: "background-color: #f3f2f1; border: 2px solid #6f777b" %>
```

Comment:
I know this is a dev view, but please no inline styles. Better leave it "ugly" than polute with inline styles ;)